clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -823,4 +824,36 @@ message Highlight {
     repeated string fields = 2;
     // Map of field name to highlight settings for field, overrides request level highlight settings
     map<string, Settings> field_settings = 3;
+}
+
+// Specify how to highlight matched text in SearchRequest.
+message HighlightV2 {
+
+    message Settings {
+        // Used along with post_tags to specify how to wrap the highlighted text.
+        repeated string pre_tags = 2;
+        // Used along with pre_tags to specify how to wrap the highlighted text.
+        repeated string post_tags = 3;
+        // Number of characters in highlighted fragment, 100 by default.
+        google.protobuf.UInt32Value fragment_size = 4;
+        // Maximum number of highlight fragments to return, 5 by default. If set to 0 returns entire text as a single fragment ignoring fragmentSize.
+        google.protobuf.UInt32Value max_number_of_fragments = 5;
+        // Specify a query here if highlighting is desired against a different query than the search query.
+        Query highlight_query = 6;
+        // Set to true to highlight fields only if specified in the search query.(Default is false)
+        google.protobuf.BoolValue field_match = 7;
+        // Sorts highlighted fragments by score when set to true. By default, fragments will be output in the order they appear in the field. (Default is false)
+        google.protobuf.BoolValue score_ordered = 8;
```

Comment:
Can we have the default behavior as score_ordered?

Reply:
order = none is default in ES, I was trying to match it. However, v1 default is true, v2 default is false introduces confusion to clients. I will change it to be true for both